nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -368,6 +368,8 @@
 
 - The `shadowstack` hardening flag has been added, though disabled by default.
 
+- `xxd` is now provided by the `tinyxxd` package, rather than `vim`, to reduce closure size and vulnerability impact. The canonical way to reference it across all platforms is `unixtools.xxd`.
```

Comment:
This is a pretty inside‐baseball change and the two should be 1:1 replacements. I don’t think we should clutter the release notes with this. (Plus, we `inherit` it from `unixtools`, so isn’t `pkgs.xxd` just as canonical?)

Reply:
I mostly copied this from https://github.com/NixOS/nixpkgs/pull/254766, which had an even more inside baseball release note.

This note I think _does_ need to be specified, because _I_ would be surprised for about 45 seconds if `xxd` wasn't connected to Vim.

Reply:
I decided to keep this.

Reply:
Fair enough, we can have the note, but I think we should drop the “canonical” sentence or at least change it to just `xxd`, because `unixtools` is somewhere between silly and an implementation detail IMO. The in‐tree users seem to generally use `xxd`, and that seems correct to me because it makes overrides easier.